projects/plugins/beta/docs/mu-plugin-info.md

```diff
@@ -0,0 +1,30 @@
+## Mu-plugin support
+
+WordPress does not provide user interface for installing and uninstalling mu-plugins. Further, mu-plugins that aren't already a single file need a "loader" that is a single file placed in the mu-plugins directory (generally `wp-content/mu-plugins/`). While there are many ways this can be done, Jetpack Beta Tester makes some assumptions about how mu-plugins it supports are installed:
+
+* The directory of code for the mu-plugin is located in the mu-plugins directory and named after the plugin slug, similar to how plugins are installed. So, for example, a plugin with slug "wpcomsh" would be at `wp-content/mu-plugins/wpcomsh/`.
+* The corresponding loader file is named with the plugin slug with `-loader.php` appended. So, for example, a plugin with slug "wpcomsh" would have its loader named `wp-content/mu-plugins/wpcomsh-loader.php`.
+
+If your installation of the mu-plugin does not follow these conventions, attempting to use Jetpack Beta Tester to install development versions of the mu-plugin may result in multiple versions of the plugin being active simultaneously, which will very likely break your site.
+
+Also, be aware that Jetpack Beta Tester will not enable a stable version of a mu-plugin for you. You will have to follow the plugin's instructions for creating the loader to properly enable it.
+
+Jetpack Beta Tester will not auto-update mu-plugins, nor will it prompt when updates are available. WordPress core's plugin update infrastructure does not handle mu-plugins, so we can't hook into it like we do normal plugins.
+
+When a development version is active and no stable version directory exists, a "Deactivate" button will be provided in the UI to easily deactivate the development version.
+If a stable directory does exist, only the usual button to activate it will be provided; if you want to deactivate the mu-plugin in this situation, do so in the normal manner by removing the loader file.
```

Comment:
Was a newline intended here? As written, line 14 and 15 render as one paragraph.

Reply:
I do that sometimes to make the markdown a little more legible when reading it as source.